src/System.Diagnostics.EventLog/src/System/Diagnostics/Reader/EventLevel.cs

```diff
@@ -0,0 +1,102 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+/*============================================================
+**
+**
+** Purpose:
+** This public class describes the metadata for a specific Level
+** defined by a Provider. An instance of this class is obtained from
+** a ProviderMetadata object.
+**
+============================================================*/
+
+using System.Collections.Generic;
+
+namespace System.Diagnostics.Eventing.Reader
+{
+    /// <summary>
+    /// Describes the metadata for a specific Level defined by a Provider.
+    /// An instance of this class is obtained from a ProviderMetadata object.
+    /// </summary>
+    public sealed class EventLevel
```

Comment:
This and eventRecord class has the same structure, will it make sense to have one common class. and both of these classes using that class for all the operations

cc @danmosemsft